### DISCUSSION TITLE
Failure to autocomplete on .md files

### DISCUSSION THREAD

**Discussion by 96312** [edheltzel/nunjucks-for-visual-studio-code#16] [Q&A]
Very useful stuff @edheltzel. I just installed this and used this on files ending with `.html` but for some reason the autocomplete is not working on markdown files in the same directory?

Is there any way to fix this?

**Comment by edheltzel** [comment]
@96312 Autocomplete or Snippets? 

When I open up an 11ty project that uses Nunjucks and start editing a markdown file the snippets work as expected...  I do have ` "editor.snippetSuggestions": "top",` set in my VSCode settings.json 

![Screenshot 2024-02-05 at 9 53 32 AM](https://github.com/edheltzel/nunjucks-for-visual-studio-code/assets/402910/f3d86d40-3a5d-4e77-af25-9abc36b98c8e)

**Comment by 96312** [comment]
That's a shame. Yeah, the snippet thing. It works on .html files, well here's a screenshot of my screen

![image](https://github.com/edheltzel/nunjucks-for-visual-studio-code/assets/34262180/abbe351b-2d3c-45e5-a479-189a60c6c17a)

@edheltzel  let me try adding that setting you suggested.

**Reply by 96312** [reply]
![image](https://github.com/edheltzel/nunjucks-for-visual-studio-code/assets/34262180/402cc370-732d-44c9-86e7-99e5fba420c8)


Here's what happens when I rename the file to `.html.` you see it works alright. I also just added that snippet to the vscode settings but had no change. @edheltzel

**Reply by edheltzel** [reply]
Do you have any other extensions for markdown, that could be changing the grammar?

**Reply by 96312** [reply]
None whatsoever. But anyways it isn't that big of a deal, I can do just fine without it just thought you should know. Closing the discussion now cheers mate
